canonical/ubuntu.com
# Update redirects from lp:ubuntu-website-content

Convert [the live redirects file](https://bazaar.launchpad.net/~canonical-website-editors/ubuntu-website-content/trunk/view/head:/redirects.txt) to JSON using [this script](https://gist.github.com/nottrobin/4a4f0162be577703cf851fbbe77f411b), and overwrite the existing `redirects.json` with these new redirects.
## QA
- Check the new redirects against [the current file](https://bazaar.launchpad.net/~canonical-website-editors/ubuntu-website-content/trunk/view/head:/redirects.txt) to check they all seem to be there.
- Load the site locally and check as many of the redirects work as you have patience for.
